app/org/sagebionetworks/bridge/play/controllers/SurveyController.java

```diff
@@ -69,42 +70,15 @@ public Result getAllSurveysMostRecentlyPublishedVersion() throws Exception {
         return okResult(surveys);
     }
     
-    public Result getSurveyForUser(final String surveyGuid, final String createdOnString) throws Exception {
-        final UserSession session = getAuthenticatedAndConsentedSession();
-
-        ViewCacheKey<Survey> cacheKey = viewCache.getCacheKey(Survey.class, surveyGuid, createdOnString); 
-        String json = viewCache.getView(cacheKey, new Supplier<Survey>() {
-            @Override public Survey get() {
-                StudyIdentifier studyId = session.getStudyIdentifier();
-                long createdOn = DateUtils.convertToMillisFromEpoch(createdOnString);
-                GuidCreatedOnVersionHolder keys = new GuidCreatedOnVersionHolderImpl(surveyGuid, createdOn);
-
-                Survey survey = surveyService.getSurvey(keys);
-                verifySurveyIsInStudy(session, studyId, survey);
-                return surveyService.getSurvey(keys);
-            }
-        });
-        return ok(json).as(JSON_MIME_TYPE);
-    }
-
-    public Result getSurveyMostRecentlyPublishedVersionForUser(final String surveyGuid) throws Exception {
```

Comment:
Although this method doesn't need to be cached, other methods in the controller do cache so the mocks set up the caching mock. What do you want me to do here?